packages/block-library/src/image/edit.native.js

```diff
@@ -256,6 +263,14 @@ class ImageEdit extends React.Component {
 			</BottomSheet>
 		);
 
+		const getImageViewer = () => (
+			<ImageViewer
+				isVisible={ this.state.showImageViewer }
+				onClose={ onImageViewerClose }
+				url ={ url }
```

Comment:
linter is complaining about the space here

Reply:
Fixed in 97df6948f422ccacfd4997b0587907f41a89d0ae